ethcore/src/block.rs

```diff
@@ -625,36 +625,19 @@ mod tests {
 			.collect();
 		let transactions = transactions?;
 
-		{
-			if ::log::max_level() >= ::log::Level::Trace {
-				let s = State::from_existing(db.boxed_clone(), parent.state_root().clone(), engine.account_start_nonce(parent.number() + 1), factories.clone())?;
-				trace!(target: "enact", "num={}, root={}, author={}, author_balance={}\n",
-					header.number(), s.root(), header.author(), s.balance(&header.author())?);
-			}
-		}
-
-		let mut b = OpenBlock::new(
+		enact(
```

Comment:
Why not just have `enact_verified()` create a new `OpenBlock` for `enact()` to use instead of just forwarding all those arguments?

Reply:
What would that archieve?  what we should do is remove enact_verified and use enact() directly,  there is only 1 place using enact_verifired anyway.

Reply:
Yeah sure, removing `enact_verified()` sounds fine as well since it's only being called once

Reply:
done